Can't build anymore on Ubuntu 20.04

Since a couple of days, I get a linking error when trying to build on Ubuntu:
```
...
Scanning dependencies of target collect
Scanning dependencies of target location
[ 48%] Building C object src/libs/CMakeFiles/collect.dir/collect.c.o
[ 48%] Building C object src/libs/CMakeFiles/location.dir/location.c.o
/usr/bin/ld: libdarktable.so: undefined reference to `dt_atomic_get_int'
/usr/bin/ld: libdarktable.so: undefined reference to `dt_atomic_set_int'
collect2: error: ld returned 1 exit status
make[2]: *** [src/CMakeFiles/darktable.dir/build.make:173: src/darktable] Error 1
make[1]: *** [CMakeFiles/Makefile2:2324: src/CMakeFiles/darktable.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 48%] Linking C shared module libimage_infos.so
[ 50%] Linking C shared module libfilmstrip.so
[ 50%] Linking C shared module libprint.so
[ 50%] Linking C shared module libmap.so
...
```
Investigating a bit, I though it could be a missing `#include "common/atomic.h"` in develop.h, but nope.

Any clue ? Thanks.

--- a/src/develop/develop.c
+++ b/src/develop/develop.c
@@ -24,6 +24,7 @@
 #include <strings.h>
 #include <unistd.h>
 
+#include "common/atomic.h"
 #include "common/debug.h"
 #include "common/history.h"
 #include "common/image_cache.h"

